build.sh

```diff
@@ -229,6 +229,10 @@ source ./config
 TOOLCHAIN_DIR=$1
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
+if [ -L "${TOOLCHAIN_DIR}" ] ; then
+  echo "The destination directory must not be a symbolic link"
+  exit 1
+fi
```

Comment:
Lets fix the check here instead:
https://github.com/osquery/osquery-toolchain/blob/d91c40e3a295a96cd96626e04273e719d31107d0/build.sh#L439

I would save the real sysroot path outside of the loop with `realpath $SYSROOT`, then use the resulting variable instead of `$SYSROOT`.